addons/actions/src/preset/addArgs.ts

```diff
@@ -0,0 +1,54 @@
+import { ParameterEnhancer, combineParameters } from '@storybook/client-api';
+import { Args } from '@storybook/addons';
+
+import { action } from '../index';
+
+// interface ActionsParameter {
+//   disable?: boolean;
+//   args?: string[];
+//   argTypesRegex?: RegExp;
+// }
+
+/**
+ * Automatically add action args for argTypes whose name
+ * matches a regex, such as `^on.*` for react-style `onClick` etc.
+ */
+export const inferActionsFromArgTypes: ParameterEnhancer = context => {
+  const { args, actions, argTypes } = context.parameters;
+  if (!actions || actions.disable || !actions.argTypesRegex || !argTypes) {
```

Comment:
Does this mean you need to ensure that this enhancer runs after docs'? That seems like a bit of a gotcha. We should document it at least

Reply:
Updated docs